front/src/sideBard/LoginForm.js

```diff
@@ -32,12 +29,26 @@ export default class LoginForm extends Component {
     }
   }
 
-  authorizeUser() {
+  async authorizeUser() {
     let data = JSON.stringify(this.state);
-    fetch("http://localhost:8000/api-token/login", {
+    let response = await fetch("http://localhost:8000/api-token/login", {
       method: "POST",
-      body: data
+      body: data,
+      headers: {
+        "Content-type": "application/json"
+      }
     });
+    let status = response.status;
+    if (status === 400){
+      console.log("jeb sie");
+    } else if (status === 200){
+      let json = await response.json();
+      localStorage.setItem("muzi_muzi_token", json.token);
+    }
+
+      // .then(response => response.json())
```

Comment:
Do we need this? I suggest to remove unused code

Reply:
Yes, very good idea, I will push a commit with a fix.

Reply:
Um, I think that was from an older commit because its not present in the HEAD commit. Issue resolved.